app/api/blocks/search/route.ts

```diff
@@ -0,0 +1,36 @@
+import { eq } from "drizzle-orm"
+import { NextRequest } from "next/server"
+import { isHash } from "viem"
```

Comment:
Good call, perhaps separately we implement the viem utility over in getBlockValueType

https://github.com/ethproofs/ethproofs/blob/af522e8e1482038a617fd162f85ae356574850d4/lib/blocks.ts#L38